spack/packages/dla-future/package.py

```diff
@@ -76,9 +76,10 @@ class DlaFuture(CMakePackage, CudaPackage, ROCmPackage):
     depends_on("whip +rocm", when="+rocm")
 
     depends_on("rocblas", when="+rocm")
-    depends_on("rocprim", when="+rocm")
     depends_on("rocsolver", when="+rocm")
-    depends_on("rocthrust", when="+rocm")
+
+    depends_on("rocprim", when="@:0.3 +rocm")
+    depends_on("rocthrust", when="@:0.3 +rocm")
```

Comment:
?

```suggestion
    depends_on("rocprim", when="@:0.3.1 +rocm")
    depends_on("rocthrust", when="@:0.3.1 +rocm")
```

Reply:
No, patch releases are for bug fixing, and adding and removing dependencies is not part of it. (Note that a version change of a dependency can be included in a patch release)
Note that v0.3.1 is not a master release, but it branched out (version_0.3) and only includes bug-fixing PRs.

Reply:
Ok, so next release will be for sure `0.4.0`, right?

Reply:
Can be `0.3.2`, but without #1045.